catalog/transformer/catalog-transformer-tika-input/src/main/java/ddf/catalog/transformer/input/tika/TikaInputTransformer.java

```diff
@@ -608,6 +608,7 @@ private void registerService(BundleContext bundleContext) {
     }
 
     mimeTypes.add("image/jp2");
+    mimeTypes.add("application/vnd.ms-visio.viewer");
```

Comment:
We should probably update the documentation as well, in [complete list of file types](https://github.com/codice/ddf/blob/4498fb9572a49df1bc3c78e9fed9eec2ca8dbd84/distribution/docs/src/main/resources/content/_metadataAttributes/complete-list-file-types.adoc).